### ISSUE TITLE
Add rfopower CPU node_exporter to Prometheus/Grafana server config

### ISSUE THREAD

**Issue by georgeloyer** [vmoa/rfopower#3]
The rfopower server (a Raspberry Pi installed on the hallway passage from the classroom to the west wing at RFO) is running the Grafana/Prometheus server for RFO and should be monitored itself so that we have a record of its uptime and performance as input for troubleshooting issues with Grafana and Prometheus. There is documentation available for a node_exporter for Linux that should work.

**Comment by georgeloyer** [comment]
Found this helpful blog page for implementing node_exporter on the Raspberry Pi 4 (rfopower):
[Prometheus Node_exporter on Raspberry Pi: How to Install](https://linuxhit.com/prometheus-node-exporter-on-raspberry-pi-how-to-install/)

1. Install the tarball specific to your RPi version. Rfopower is an RPi4. For others, consult the [project releases page](https://github.com/prometheus/node_exporter/releases) for node_exporter.
`wget https://github.com/prometheus/node_exporter/releases/download/v0.18.1/node_exporter-0.18.1.linux-armv7.tar.gz`

2. Un-tar the tarball in a convenient location
`tar -xvzf node_exporter-0.18.1.linux-armv7.tar.gz `

3. Copy it into a standard location for execution:
`sudo cp node_exporter-0.18.1.linux-armv6/node_exporter /usr/local/bin `
Change the file to be executable if needed (I found the the file was already executable in the tarball)

4. Create a service account for node_exporter
`sudo useradd -m -s /bin/bash node_exporter `

5. Create a directory in /var/lib for node_exporter, the service, to use:
`sudo mkdir /var/lib/node_exporter
chown -R node_exporter:node_exporter /var/lib/node_exporter `

6. Set up node_exporter to run via systemctl. Set up the node_exporter.service unit file:
`sudo vi /etc/systemd/system/node_exporter.service`
And then add the following text:

```
[Unit]
Description=Node Exporter

[Service]
# Provide a text file location for https://github.com/fahlke/raspberrypi_exporter data with the
# --collector.textfile.directory parameter.
ExecStart=/usr/local/bin/node_exporter --collector.textfile.directory /var/lib/node_exporter/textfile_collector

[Install]
WantedBy=multi-user.target
```

7. Reload systemd, then enable and start node_exporter service:
`sudo systemctl daemon-reload 
sudo systemctl enable node_exporter.service
sudo systemctl start node_exporter.service `

8. Test the output of the service by opening a browser page: [rfopower metrics](http://192.168.74.11:9100/metrics). Port 9100 is the standard port used by node_exporter and prometheus.

9. Edit the prometheus.yml file on the rfopower server at /opt/prometheus/prometheus.yml. Add the following configuration text to that file:
```
- job_name: 'rfopowercpu'
    static_configs:
    - targets: ['192.168.74.11:9100']
```

**Comment by georgeloyer** [comment]
Grafana has been configured so that additional jobs should just come up in the appropriate dashboard (testing TBD).  For Linux servers, like the one in this issue, the public dashboard from Grafana, Node Exporter Full (ID: 1860) automatically reads the data imported by prometheus by the node and displays it in the dashboard. Additional Linux nodes will be added as options in the drop down of the dashboard to choose.  For Windows servers, the public dashboard from Grafana, Windows Exporter Dashboard (ID: 15974) does the same for the Eagle and other Windows servers once node_exporter has been configured and is running as a service.

To experiment with other dashboards, click on the Dashboard icon in Grafana's home screen, click on New, and then Import.  Enter the ID into the ID field and wait for the data to populate (and the JSON to be uploaded automatically). Choose the data source: PowerPi is our Prometheus data source, and then Save.  The dashboard will be installed and opened. If the new dashboard is working properly and is working for the standard node_exporter data for our Linux or Windows installations, it just works.  I tried several that didn't work or didn't work until I had manually tweaked the queries either in the dashboard or in the individual panels in the dashboard.  The two noted above just worked.
